bears/r/FormatRBear.py

```diff
@@ -56,25 +56,33 @@ def create_arguments(filename, file, config_file,
             Maximum number of characters for an expression.
 
             Example: If ``20`` then
-            ``1 + 1 + 1 + 1 + 1 + 1 + 1`` changes to
-            ``1 + 1 + 1 + 1 + 1 + 1 +``
-            ``    1``
+            '''
+            1 + 1 + 1 + 1 + 1 + 1 + 1
+            '''
+             changes to
+            '''
+            1 + 1 + 1 + 1 + 1 + 1 +
+                1
+            '''
         """
         rcode = ('library(formatR);'
                  'formatR::tidy_source('
                  'source="' + escape(filename, '"\\') + '",'
                  'comment={r_keep_comments},'
                  'blank={r_keep_blank_lines},'
                  'arrow={r_use_arrows},'
-                 'width.cutoff={r_max_expression_length},'
                  'brace.newline={r_braces_on_next_line},'
-                 'indent={tab_width}'
-                 ')'.format(tab_width=tab_width,
-                            r_keep_comments=_map_to_r_bool(r_keep_comments),
-                            r_keep_blank_lines=_map_to_r_bool(
-                                r_keep_blank_lines),
-                            r_use_arrows=_map_to_r_bool(r_use_arrows),
-                            r_braces_on_next_line=_map_to_r_bool(
-                                r_braces_on_next_line),
-                            r_max_expression_length=r_max_expression_length))
+                 'indent='
+                 '{tab_width}'.format(tab_width=tab_width,
+                                      r_keep_comments=_map_to_r_bool(
+                                          r_keep_comments),
+                                      r_keep_blank_lines=_map_to_r_bool(
+                                          r_keep_blank_lines),
+                                      r_use_arrows=_map_to_r_bool(
+                                          r_use_arrows),
+                                      r_braces_on_next_line=_map_to_r_bool(
+                                          r_braces_on_next_line)))
+        # Disable r_max_expression_length if it is equal to 0
+        rcode += ('' if r_max_expression_length == 0
+                  else ',width.cutoff=' + str(r_max_expression_length)) + ')'
```

Comment:
I forgot to say that you maybe want to disable `r_max_expression_length` by default (so setting inside the signature `r_max_expression_length = 0`) ^^